src/IO/ReadWriteHelpersCommon.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <cstring>
+#include <string>
+#include <string_view>
+#include <span>
+#include <cstddef>
+
+#include <Common/StackTrace.h>
+#include <Common/formatIPv6.h>
+#include <Common/DateLUT.h>
+
+#include <Common/LocalDate.h>
+#include <Common/LocalDateTime.h>
+#include <Common/LocalTime.h>
+#include <Common/transformEndianness.h>
+#include <base/find_symbols.h>
+#include <base/StringRef.h>
```

Comment:
[minor] do we need this headers here (transformEndiannes, find_symbols and StringRef)?

Reply:
Personally I try to follow IWYU (include-what-you-use), therefore I'd prefer them to have in the readhelpers and writehelpers separately even if they are shared. But it's a more personal style preference.